src/main/java/io/elastic/jdbc/providers/TimeStampColumnNamesProvider.java

```diff
@@ -0,0 +1,54 @@
+package io.elastic.jdbc.providers;
+
+import io.elastic.api.SelectModelProvider;
+import io.elastic.jdbc.utils.Utils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+import java.sql.*;
```

Comment:
Could you please specify only classes you use instead of '.*'?

Reply:
I've tried, but in this case, it is needed `*`